addons/project/static/tests/tours/personal_stage_tour.js

```diff
@@ -32,7 +32,7 @@ tour.register('personal_stage_tour', {
     trigger: '.o_column_quick_create .o_quick_create_folded'
 }, {
     content: "Create a new personal stage",
-    trigger: 'input[placeholder="Column title"]',
+    trigger: 'input[class="form-control o_input"]',
```

Comment:
Equivalent to: 
```suggestion
    trigger: 'input.form-control.o_input',
```
🙂